pybikes/data/callabike.json

```diff
@@ -3,10 +3,10 @@
         {
             "tag": "callabike-aschaffenburg",
             "meta": {
-                "latitude": 49.436467,
```

Comment:
Good point. I made a few tries: The lat/long seems to be the initial center of the search. This changes if you type an address into the search mask. So you are right: you will not get all stations with one request (DAMN!).
I think I will not have so much time to make changes at the moment. But feel free to play around with my code. Hopefully it is at least a good basis.